__tests__/skpm/basic.test.js

```diff
@@ -2,6 +2,10 @@ import * as React from 'react';
 import * as sketch from 'sketch'; // eslint-disable-line
 import { render, View, Artboard, Text } from '../../src';
 
+function getDoc(context, document) {
```

Comment:
Could you add a comment regarding what this function is doing, and why it is necessary?